package.json

```diff
@@ -47,6 +47,8 @@
     "typescript": "^4.5.4"
   },
   "dependencies": {
+    "cheminfo-types": "^0.9.1",
+    "jest-matcher-deep-close-to": "^3.0.2",
```

Comment:
this should be in devDependencies (`npm i --save-dev jest-matcher-deep-close-to`)

Reply:
ah right I will fix this right away